### DISCUSSION TITLE
Temp control

### DISCUSSION THREAD

**Discussion by gerben-f1** [theyosh/TerrariumPI#959] [Q&A]
Hello there,

who can help me in te good direction.
I have terrariumpi running with 4 channel dimmer.
Now i have my heating pad connected to channel 1 from dimmer.

how can set it up so it stays 35 degrees daytime
And 22 degrees night time

mesurement is done by temp sensor.

**Comment by theyosh** [comment]
Ok, lets see if I can explain it.

First setup your configuration during the **day** time. This is important.

Set the temperature sensor at a min alarm value of 30 and a max alarm value of 40. This will result in an average temperature of 35 degrees. Make sure that the limit min value is about 10 and the limit max about 50 or more.

Add the dimmer as an relay to the system.

Now create first an enclosure. Give it a name and select doors or webcam when available/needed. 

And than create a light area, where you specify when the main lights are on. This is needed in order to determine day or night period for lowering the temperature. You can use a fixed time or use weather source. But you need to setup a lights area with the toggle 'main lights' on. https://theyosh.github.io/TerrariumPI/setup/#main-lights

Next create a heating area. Fill out the form and select temperature mode and select the temperature sensor that is configured with the correct temperature values. Then select the dimmer relay in the relays tabs.
Do **not** fill in the fields in the row with Begin time, end time etc. https://theyosh.github.io/TerrariumPI/setup/#other-areas
And fill in a value of 13 (35-22) in the Day/night difference field. This will lower the average temperature during the night.

Save the area and it should be done.

As you are using a dimmer, it will take some time to get at the average values of 35 and 22. But I expect after an hour, it should stay around the average values.

**Reply by gerben-f1** [reply]
I added the dimmer as a switch, that is not correct?

i have the i2c 4channel led AC dimmer

**Reply by theyosh** [reply]
Do not understand the question. When you add a relay, you pick your hardware type. And that makes it a dimmer or not.

If you have added this relay already, in the relay listing you should see 90% circle with a number inside. If that is the case, you have a dimmer. But for you setup it does not matter.

**Reply by gerben-f1** [reply]
I have this so its good i think
![IMG_0288](https://github.com/user-attachments/assets/bca42e81-e022-47a4-9db0-b671bf6e91a6)

**Reply by theyosh** [reply]
Yup, dat is correct. Dit is een dimmer functie. En het kan in NL. Is misschien wat makkelijker communiceren.

**Reply by gerben-f1** [reply]
Goedendag,

merk toch een fout op.
Precies gedaan zoals u zei.
Maar temperatuur veranderd niet goed.
Het is nu dag maar dan moet de temp eigenlijk 35graden zijn. Maar geeft nu tussen 17 en 27 aan
![IMG_0292](https://github.com/user-attachments/assets/dd8dfff7-9bde-44b5-9fde-683570f52a26)

**Comment by gerben-f1** [comment]
Thanks for the help.
It works.

**Comment by gerben-f1** [comment]
Zo is de instelling van de dimmer
![IMG_0294](https://github.com/user-attachments/assets/27cc4713-567e-42e9-9efe-1cdf93d89676)
![IMG_0293](https://github.com/user-attachments/assets/1d056d89-93d0-426c-8cb3-622afc991a5f)

**Reply by theyosh** [reply]
Ik denk dat je 'Dag/nacht bron' op verlichting moet zetten. Nu is het onbekend, en werkt het niet.

Als je dat gedaan hebt, weet ik zo niet of de temperatuur vanzelf omhoog gaat. Als dat na 5 minuten niet gebeurd, dan zul je zelf weer de sensoren op de oude, overdag temperatuur.

**Reply by gerben-f1** [reply]
Heb het ingesteld ga het in de gaten houden.
Voor zover super bedankt.
